HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHighPtTripletStepSequence_cfi.py

```diff
@@ -30,3 +30,24 @@
 
 from Configuration.ProcessModifiers.seedingLST_cff import seedingLST
 (seedingLST & trackingLST).toReplaceWith(HLTHighPtTripletStepSequence, _HLTHighPtTripletStepSequenceLSTSeeding)
+
+from ..modules.hltMkFitSiPixelHits_cfi import *
+from ..modules.hltMkFitSiStripHits_cfi import *
+from ..modules.hltMkFitSiPhase2Hits_cfi import *
+from ..modules.hltMkFitEventOfHits_cfi import *
+from ..modules.hltHighPtTripletStepTrackCandidatesMkFitSeeds_cfi import *
+from ..modules.hltHighPtTripletStepTrackCandidatesMkFit_cfi import *
+
+_HLTHighPtTripletStepSequenceMkFit = cms.Sequence(
+    hltMkFitSiPixelHits
+    +hltMkFitSiPhase2Hits
+    +hltMkFitEventOfHits
+    +hltHighPtTripletStepTrackCandidatesMkFitSeeds
+    +hltHighPtTripletStepTrackCandidatesMkFit
+    +hltHighPtTripletStepTrackCandidates
+    +hltHighPtTripletStepTracks
+    +hltHighPtTripletStepTrackCutClassifier
+    +hltHighPtTripletStepTrackSelectionHighPurity
+)
+from Configuration.ProcessModifiers.trackingMkFitHighPtTripletStep_cff import hltTrackingMkFitHighPtTripletStep
+(hltTrackingMkFitHighPtTripletStep & trackingLST).toReplaceWith(HLTHighPtTripletStepSequence,_HLTHighPtTripletStepSequenceMkFit)
```

Comment:
why is this conditioned on LST? shouldn't we support the two-iter no-LST case as well?
```suggestion
hltTrackingMkFitHighPtTripletStep.toReplaceWith(HLTHighPtTripletStepSequence,_HLTHighPtTripletStepSequenceMkFit)
```